src/pyorderly/outpack/location_driver.py

```diff
@@ -14,10 +14,10 @@ class LocationDriver(AbstractContextManager):
     """
 
     @abstractmethod
-    def list(self) -> Dict[str, PacketLocation]: ...
+    def list(self) -> dict[str, PacketLocation]: ...
 
     @abstractmethod
-    def metadata(self, packet_ids: List[str]) -> Dict[str, str]: ...
+    def metadata(self, packet_ids: builtins.list[str]) -> dict[str, str]: ...
```

Comment:
Quite surprisingly, the `list` method is actually in scope in the method signature (but not in its body, where `self.list()` must be used). This shadows the built-in definition, hence the explicit call here. ruff was clever enough to figure this out automatically.

Reply:
It might be worth renaming the `list` method to `list_packets` at some point to avoid this ambiguity